src/components/SiteHeader/HeaderLink.tsx

```diff
@@ -1,36 +1,67 @@
 import classNames from 'classnames'
-import { Link } from 'gatsby'
+import Link from 'gatsby-link'
 import _ from 'lodash'
 import React, { ReactNode } from 'react'
+import { Text } from 'rebass/styled-components'
+import styled from 'styled-components'
+
+import { HeaderTheme } from 'components/SiteHeader/SiteHeader'
+import useHover from 'utils/use-hover'
+
+const StyledHeaderLink = styled(Text)`
+  transition: color 0.3s;
+`
+
+const getIsActivePath = _.memoize(
+  ({ href, pathname }): boolean =>
+    (href === '/' && pathname === '/') ||
+    (href !== '/' && _.startsWith(pathname, href)),
+)
+
+const getThemeForPathname = (pathname: string): HeaderTheme =>
+  _.startsWith(pathname, '/photography') ? HeaderTheme.LIGHT : HeaderTheme.DARK
 
 interface Props {
-  children?: ReactNode
+  children: ReactNode
   className?: string
   href: string
-  isActive?: boolean
-  pathname?: string
+  pathname: string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)